packages/http-client-java/generator/http-client-generator-clientcore-test/src/main/java/encode/bytes/HeaderClient.java

```diff
@@ -47,7 +47,7 @@ public final class HeaderClient {
     @Metadata(properties = { MetadataProperties.GENERATED })
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Response<Void> defaultMethodWithResponse(byte[] value, RequestContext requestContext) {
-        return this.instrumentation.instrumentWithResponse("Header.default", requestContext,
+        return this.instrumentation.instrumentWithResponse("BytesClient.default", requestContext,
```

Comment:
What do we want to have in this text? Is `BytesClient.##` good enough? Or we rather want to have `HeaderClient.##`?

If latter, I could help see if there is someway to get the e.g. `HeaderClient`.

I am fine with the code. If you want to do a follow-up PR to refine the logic, I am also fine.

Reply:
Since this results in BytesClient.default for all of QueryClient, PropertyClient, HeaderClient and *BodyClient types, we may need to be a bit more specific. One option discussed with @lmolkova  was to use the crossLanguageDefinitionId but we need to confirm this can be public facing and ensure this doesn't break.